HelpSource/Help.schelp

```diff
@@ -2,38 +2,266 @@ TITLE::Help
 summary::Documentation home
 categories::Help
 
-SuperCollider is an audio server, programming language, and IDE for sound synthesis and algorithmic composition.
-
 NOTE:: link::Guides/News-3_13##News in SuperCollider version 3.13:: ::
 
-SECTION::Search and browse
+
+SECTION:: An introductory overview
+
+*SuperCollider* is a cross-platform environment for audio synthesis and algorithmic composition used by musicians, artists and researchers working with sound, with three main components:
+
 definitionlist::
-## link::Search:: || Search all documents and methods
-## link::Browse:: || Browse all documents by categories
+## soft:: ::• soft:: ::link::Classes/Server:: – a real-time audio engine.||
+The default server is link::Classes/Server#Switching the server application#scsynth::. To take full advantage of multi-core, an alternative sound engine, link::Classes/Server#Switching the server application#supernova::, can be used.
+
+## soft:: ::• soft:: ::Client soft:: ::– sclang. a text-based programming language and interpreter. ||
+
+emphasis::sclang:: controls audio synthesis processes on the server and offers an link::Overviews/JITLib##interactive programming environment for on-the-fly coding::, link::Tutorials/A-Practical-Guide/PG_01_Introduction##an extensive library for creating patterns:: and link::Classes/Scale##a library of scales and modes from various cultures::.
+link::Tutorials/Getting-Started/15-Sequencing-with-Routines-and-Tasks##Creating processes that unfold over time:: is straighforward in sclang.
+It also provides link::Guides/GUI-Introduction##all the essential features of a graphical user interface:: that can be used to create custom user interfaces and data visualization.
+With support for
+link::Guides/UsingMIDI##MIDI::,
+link::Guides/OSC_communication##OSC:: and
+link::Guides/Working_with_HID##HID::,
+sclang can be easily interfaced with local and networked hardware and software.
+
+## soft:: ::• soft:: ::link::Classes/ScIDE:: soft:: ::– a dedicated editor for SuperCollider with an integrated help system.||
+link::Guides/SCIde##The SuperCollider-IDE (Integrated Development Environment) :: allows you to start writing code right out of the box. Community members also maintain sclang integration with other popular IDEs such as link::https://github.com/scztt/vscode-supercollider##Visual Studio Code, VSCodium::, link::https://github.com/capital-G/sc_kernel##Jupyter::, link::https://github.com/davidgranstrom/scnvim##SCNvim::, link::https://github.com/supercollider/scvim##scvim::, link::https://github.com/supercollider/scel##Emacs::, and so on.
```

Comment:
```suggestion
The SuperCollider Integrated Development Environment (link::Guides/SCIde##IDE::) allows you to start writing code right out of the box. Community members also maintain sclang integration with other popular IDEs such as link::https://github.com/scztt/vscode-supercollider##Visual Studio Code, VSCodium::, link::https://github.com/capital-G/sc_kernel##Jupyter::, link::https://github.com/davidgranstrom/scnvim##SCNvim::, link::https://github.com/supercollider/scvim##scvim::, link::https://github.com/supercollider/scel##Emacs::, and so on.
```

Reply:
I think it’s ok to list alternative IDEs, but probably best to avoid linking to repos outside of SC's GitHub organization. It would be best to link to a wiki on the SC repo that has a list of IDEs with SC support. That way it can be maintained more dynamically without the need to update the help docs (which can take 6+ months between updates).

Reply:
> It would be best to link to a wiki on the SC repo that has a list of IDEs with SC support.

this is a very good idea! I could not find a page on a wiki. Where can I find it?

Reply:
well there's this: https://github.com/supercollider/supercollider/wiki/Systems-interfacing-with-SC - needs some editing perhaps!

Reply:
> https://github.com/supercollider/supercollider/wiki/Systems-interfacing-with-SC

Ah yes!  However, the Jupyter extension and the VSCode extension should be added to this page.

I can link that page. Then the following sentence
>Community members also maintain sclang integration with other popular IDEs such as link::https://github.com/scztt/vscode-supercollider##Visual Studio Code, VSCodium::, link::https://github.com/capital-G/sc_kernel##Jupyter::, link::https://github.com/davidgranstrom/scnvim##SCNvim::, link::https://github.com/supercollider/scvim##scvim::, link::https://github.com/supercollider/scel##Emacs::, and so on.

should be changed as follows?

>Community members also maintain link::https://github.com/supercollider/supercollider/wiki/Systems-interfacing-with-SC##sclang integration with other systems such as various programming languages and editors.::